samples/SkijaInjectSample/src/main/kotlin/SkijaInjectSample/SkiaPanel.kt

```diff
@@ -34,5 +34,6 @@ open class SkiaPanel: JLayeredPane {
 
      override fun removeNotify() {
         super.removeNotify()
+        layer.dispose()
```

Comment:
Should be called before `removeNotify`, otherwise it will crash on Linux

Reply:
Done.